test/auth.e2e-spec.ts

```diff
@@ -683,4 +683,24 @@ describe("AuthController e2e Tests", () => {
             expect(res.headers.location).toMatch(re);
         });
     });
+
+    describe("Initiate Github OAuth GET /auth/github/login", () => {
+        it("should redirect ", async () => {
+            const res = await request(app.getHttpServer())
+                .get("/auth/github/login")
+                .expect(302);
+
+            const clientId = Oauth.github.clientId;
```

Comment:
I see that in the beforeAll() part of the test suite, we declare a provider for discord but i don't see any for Github. The github part is declared indeed in the Oauth variable but shouldn't it also be added as a new useValue?